wordle-web/components/SettingsDialog.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-dialog v-model="modelValue">
+    <v-card class="ma-auto" min-width="400">
+      <v-sheet color="secondary">
+        <v-card-title>Settings</v-card-title>
+      </v-sheet>
+      <v-card-text>
+        <v-divider class="my-4" />
+        Light/Dark Mode:
+        <v-btn
+          color="secondary"
+          @click="toggleTheme"
+          icon="mdi-theme-light-dark" />
+        <v-divider class="my-4" />
+      </v-card-text>
+
+      <v-sheet class="ml-6" width="200">
+        <v-form @submit.prevent>
+          <v-select
+            color="secondary"
+            v-model="selectedTheme"
+            label="Select theme"
+            :items="['Regular', 'Jungle', 'Flamingo']"></v-select>
+          <v-btn
+            color="secondary"
+            @click="changeColorScheme"
+            class="mb-6"
+            type="submit"
+            >Submit</v-btn
+          >
+        </v-form>
+      </v-sheet>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script setup lang="ts">
+import { useTheme } from 'vuetify';
+import nuxtStorage from 'nuxt-storage';
+import { Theme } from '~/scripts/theme';
+
+const modelValue = defineModel<boolean>({ default: false });
+const theme = useTheme();
+const selectedTheme = ref('Regular');
+
+onMounted(() => {
+  var defaultTheme = nuxtStorage.localStorage.getData('theme');
+  theme.global.name.value = defaultTheme ?? 'regularDark';
+});
+
+function toggleTheme() {

```

Comment:
great job on getting the themes to work on either light or dark, this was one of the hardest parts for me.

Reply:
Definitely was tough. Thanks :)